packages/amplication-client/src/Settings/NewApiToken.tsx

```diff
@@ -101,11 +101,12 @@ const NewApiToken = ({ onCompleted }: Props) => {
       </div>
       <Formik
         initialValues={INITIAL_VALUES}
+        validateOnBlur={false}
         validate={(values: models.ApiTokenCreateInput) =>
           validate(values, FORM_SCHEMA)
         }
-        onSubmit={handleSubmit}
-        validateOnMount
+        onSubmit={handleSubmit}  
+        
```

Comment:
validateOnMount is false by default so the fact that removing it fixed the issue is weird 
https://formik.org/docs/api/withFormik#validateonmount-boolean
@yuval-hazaz what do you think?